### PR TITLE
Description of the event is missing with import #1872

### PR DESCRIPTION
**Pull request by supyrow**
i found an issue with my previous PR,  it was not catching correctly.

With the the additions made, importing task and event descriptions now appear to work on all of my calendars. I removed all events, then re-imported. appears correct. i use betterbird for a client and radicale for calendar server.

again this is to close issue #1872
Description of the event is missing with import #1872

this should update both Constants.kt and IcsImporter.kt

### CODE REVIEW

**Review by supyrow — COMMENTED** [review]
should be good now

File: app/src/main/kotlin/com/simplemobiletools/calendar/pro/helpers/Constants.kt [hunk 1]
```diff
@@ -159,7 +159,8 @@ const val LAST_MODIFIED = "LAST-MODIFIED"
 const val DTSTAMP = "DTSTAMP:"
 const val DURATION = "DURATION:"
 const val SUMMARY = "SUMMARY"
-const val DESCRIPTION = "DESCRIPTION:"
+const val DESCRIPTION = "DESCRIPTION"
```
Comment (Aga-C) [inline app/src/main/kotlin/com/simplemobiletools/calendar/pro/helpers/Constants.kt:162]: This change has broken the export.

Comment (supyrow) [reply app/src/main/kotlin/com/simplemobiletools/calendar/pro/helpers/Constants.kt:162]: acknowledged. 
currently working on it.

Comment (supyrow) [reply app/src/main/kotlin/com/simplemobiletools/calendar/pro/helpers/Constants.kt:162]: strike that.   export doesn't work with current master branch in the first place.  i just did a fresh install and build from current master branch.   export is already broken.  id like to fix it, however it is important to note my patch does not break it.

Comment (Aga-C) [reply app/src/main/kotlin/com/simplemobiletools/calendar/pro/helpers/Constants.kt:162]: I checked on the master branch and export is working. I mean exporting "normal text" descriptions, not HTML ones. In ICS files after your change `:` is missing after `DESCRIPTION`.

Comment (supyrow) [reply app/src/main/kotlin/com/simplemobiletools/calendar/pro/helpers/Constants.kt:162]: okay, now i am really confused.  master branch, current.  when i got to hamburger (the three dots) select "Export Events to an .ics file"     i get an error, "no entries for exporting"  if i import an ics file from hamburger, it doesnt work.    if i import from a file manger it imports.

what are you doing different??  hold my hand for a moment.

Comment (Aga-C) [reply app/src/main/kotlin/com/simplemobiletools/calendar/pro/helpers/Constants.kt:162]: I have a fresh copy of the app running on an emulator (latest master). I've created one event with description, then export from the three dots menu. On your branch, export is also working, but the `DESCRIPTION` in .ics is malformed.

Comment (supyrow) [reply app/src/main/kotlin/com/simplemobiletools/calendar/pro/helpers/Constants.kt:162]: (i am using an actual phone by the way. android 13, galaxy a71)
"On your branch, export is also working, but the DESCRIPTION in .ics is malformed."   ohh...   i get an exported file with 0 bytes.  when using master current.  my patch, i get the same result.      i wish i could get a malformed ics.  id have something to work from.

i am currently downloading the android 13 emulator/virtual device, "tiramisu" under android studio.  ( what is the standard to use???)  what is your setup for virtual/emulator 

i'd really like to see what is going on here.    it seems both our testing come back different, slightly.

in the meantime, do you want to try something?
in `helpers/IcsExporter.kt`
line 90: `writeLn("$DESCRIPTION_EXPORT$reminderLabel")`
line 121:  `out.writeLn("$DESCRIPTION_EXPORT$substring")`

in `helpers/Constants.kt`
add
`const val DESCRIPTION_EXPORT = "DESCRIPTION:"`

Comment (supyrow) [reply app/src/main/kotlin/com/simplemobiletools/calendar/pro/helpers/Constants.kt:162]: could you please????   send me an example ics with the exported ics ?? from master branch and one from my patch??

Comment (supyrow) [reply app/src/main/kotlin/com/simplemobiletools/calendar/pro/helpers/Constants.kt:162]: on the emulator, this works. and DESCRIPTION is correct in the file. from MY tests.
in `helpers/IcsExporter.kt`
line 90: `writeLn("$DESCRIPTION_EXPORT$reminderLabel")`
line 121: `out.writeLn("$DESCRIPTION_EXPORT$substring")`

in `helpers/Constants.kt`
add
`const val DESCRIPTION_EXPORT = "DESCRIPTION:"`

on the actual phone.   no dice.  i get an export file with 0 bytes.
emulator works great, i think i'll just use the emulator instead.    

i don't know what to make of this now.  any ideas, on why my phone and probably many other will not work correctly?  or is my phone the problem??  not sure what to work on.  let me know your findings, please?

Comment (Aga-C) [reply app/src/main/kotlin/com/simplemobiletools/calendar/pro/helpers/Constants.kt:162]: I've tested now both on emulator and real phone (OnePlus 8 Pro with Android 13) - export+import, in the way I've described, works. ICS file is populated, so I don't know, maybe that's some issue related to your phone - e.g. insufficient permissions? Maybe Samsung's OS somehow limits debug versions of the apps? I really don't know.
The bug I've raised is now fixed, so from my perspective, it's good. Now your PR should be confirmed by @tibbi if it's OK for him and if it solves the bug it was originally made to fix. Good luck 🙂

Comment (supyrow) [reply app/src/main/kotlin/com/simplemobiletools/calendar/pro/helpers/Constants.kt:162]: yeah it must be my phone.  i have it locked down pretty tight.   well, thank you for your help!

**Comment by tibbi** [comment]
so does this work well now?

**Comment by supyrow** [comment]
it does.

**Comment by Aga-C** [comment]
I see that you've requested a review from me, but I'm not a person to do reviews here 😉. However, I've tested your code - I've exported ICS from Outlook (Teams meeting from work) and after importing I got no description, so I assume the bug is not fixed.

File: app/src/main/kotlin/com/simplemobiletools/calendar/pro/helpers/Constants.kt (hunk 1 above)
Comment (supyrow) [reply app/src/main/kotlin/com/simplemobiletools/calendar/pro/helpers/Constants.kt:162]: > I've tested now both on emulator and real phone (OnePlus 8 Pro with Android 13) - export+import, in the way I've described, works. ICS file is populated, so I don't know, maybe that's some issue related to your phone - e.g. insufficient permissions? Maybe Samsung's OS somehow limits debug versions of the apps? I really don't know. The bug I've raised is now fixed, so from my perspective, it's good. Now your PR should be confirmed by @tibbi if it's OK for him and if it solves the bug it was originally made to fix. Good luck slightly_smiling_face

so... this is inaccurate?

**Comment by supyrow** [comment]
damnit.  i did not intend on merging that last commit.  i am unable to revert it.

anyway, I can import and export with description using my patch. 

[EventExported.ics.txt](https://github.com/SimpleMobileTools/Simple-Calendar/files/10987633/EventExported.ics.txt)
[EventImported.ics.txt](https://github.com/SimpleMobileTools/Simple-Calendar/files/10987634/EventImported.ics.txt)

**Comment by Aga-C** [comment]
I tested on my ICS file and the ICS attached to #1872 (with modified date to 2023), and both were not working. My file is very similar to the one attached to the issue you meant to fix, so you can just test on that one.

**Comment by supyrow** [comment]
ohh!   i see now.  alright, i'll keep you posted

**Comment by Aga-C** [comment]
> @Aga-C please confirm import and export works for you?

Now it works fine. I've tested a few cases and all are working properly. However, I'm not an owner of the code, so I won't do the code review for you,

**Comment by supyrow** [comment]
that is great to hear.  Thank you for your help and patience!

**Review by Aga-C — APPROVED** [review]

**Comment by tibbi** [comment]
ok, hope it wont break anything. Thanks

**Pull request merged**